index.html

```diff
@@ -52,5 +52,149 @@ <h1>Hey there. I'm <br>Victor<br><span>
         </div>
         
     </section>
+    <section class="work">
+        <div class="work-title">
+            <h2>My Recent <br>Works</h2>
+            <img src="images/Indicator.png" alt="indicator line">
```

Comment:
- [ ] Please take a look at this image right here, it is broken and not displaying normally. kindly fix it.
- [ ] Please center the arrow icon, it is a bit more aligned to the left 
<img width="391" alt="my" src="https://user-images.githubusercontent.com/79058364/159113201-4e164ec3-48c1-421f-9d3f-7fa4805fbb02.png">

Reply:
@BenMukebo I have fixed the position. Thanks.